compiler_opt/rl/inlining/gin_configs/common.gin

```diff
@@ -7,3 +7,9 @@ clang_path=None
 runners.InliningRunner.llvm_size_path=%llvm_size_path
 runners.InliningRunner.clang_path=%clang_path
 runners.InliningRunner.launcher_path=%launcher_path
+
+problem_config.flags_to_add.add_flags=()
+problem_config.flags_to_delete.delete_flags=('-split-dwarf-file','-split-dwarf-output',)
+# For AFDO profile reinjection elide overlapping flags from delete_flags and set
```

Comment:
Nit: now the comment is confusing: which overlapping flags :)

Reply:
argh, fixed in https://github.com/google/ml-compiler-opt/pull/143